.babelrc

```diff
@@ -12,6 +12,7 @@
     "@babel/plugin-transform-arrow-functions",
     "@babel/plugin-transform-function-name",
     "@babel/plugin-transform-spread",
-    "@babel/plugin-transform-parameters"
+    "@babel/plugin-transform-parameters",
+    "@babel/plugin-transform-shorthand-properties"
```

Comment:
🔥

Reply:
Are you okay with that? I think this transform produces code of reasonable size. It's no bigger than handwritten ES5. And ES6+ version is much more concise :)

Reply:
No, this is right because `cjs` cannot support it right now but definitely should